INSS.EIIR.Functions/Functions/Subscriber.cs

```diff
@@ -51,7 +51,7 @@ public class Subscriber
     [OpenApiParameter(name: "subscriberId", In = ParameterLocation.Query, Required = true, Type = typeof(string), Description = "The subscriber Id")]
     [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "text/plain", bodyType: typeof(Models.SubscriberModels.Subscriber), Description = "Subscriber details for the Id specified")]
     public async Task<IActionResult> GetSubscriberById(
-        [HttpTrigger(AuthorizationLevel.Function, "get", Route = "subscribers/{id}")] HttpRequest req)
+        [HttpTrigger(AuthorizationLevel.Function, "get", Route = "subscribers/{subscriberId}")] HttpRequest req)
```

Comment:
remove {subscriberId} as this is passed as a parameter. Becuase of this cannot use name subscribers as it used elsewhere so rename to subscriberById

Reply:
Changed subscriberId param to be route rather than query